photo.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html" charset="utf-8">
+	  <link rel="StyleSheet" href="style.css" type="text/css">
+	  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap&subset=cyrillic" rel="stylesheet">
+	  <title>Фотогалерея</title>
+  </head>
+  <body>
+    <header class="page-header">
+	    <nav class="page-header__nav nav">
+	      <div class="nav__menu-prewiew">
+	        <a class="nav__logo">
+	          <img class="nav__logo-image" src="" alt="Турагенство PINK" width="" height="">
+	        </a>
+	        <button class="nav__toggle" type="button">Меню</button>
+		    </div>
+		    <ul class="nav__list">
+		      <li class="nav__list-item"><a href="index.html" class="nav__list__link">Главная</a></li>
+		      <li class="nav__list-item"><a href="photo.html" class="nav__list__link">Фотографии</a></li>
+		      <li class="nav__list-item"><a href="form.html" class="nav__list__link">Конкурс</a></li>
+		      <li class="nav__list-item"><a href="#" class="nav__list__link">HTML Академия</a></li>
+		    </ul>
+	    </nav>
+	  </header>
+	  <main class="page-photo">
+	    <h1 class="visually-hidden">Фотогалерея</h1>
+      <section class="banner"
```

Comment:
```suggestion
      <section class="banner">
```
Будь внимательней, закрывай теги.